spec/units/magic_variables_spec.rb

```diff
@@ -22,6 +22,15 @@
         expect(magic_variables.fetch(:_prior)).to eq 'a-branch-name'
         expect(repo).to have_received(:revision_name).with('@{-1}')
       end
+
+      it 'raises when there is no prior branch' do
+        repo = double('GitRepository', revision_name: nil)
+        magic_variables = described_class.new(repo)
+
+        expect { magic_variables.fetch(:_prior) }.to raise_exception(
+          Gitsh::UnsetVariableError,
```

Comment:
We're asserting that sensible errors are output in the integration tests. I'm not sure there's much value in repeating them here.